gitcoins-cordapp/build.gradle

```diff
@@ -0,0 +1,140 @@
+buildscript {
+    ext {
+        ext.corda_release_group = 'net.corda'
```

Comment:
You don't want all of those `ext.` prefixes because these properties are already within the `ext {}` block.

Reply:
Ah yes ... I was looking at the wrong examples!